pvt/helpers/src/test/relativeError.ts

```diff
@@ -3,42 +3,49 @@ import { Decimal } from 'decimal.js';
 import { BigNumberish } from 'ethers';
 import { bn, pct } from '../numbers';
 
-export function expectEqualWithError(actual: BigNumberish, expected: BigNumberish, error: BigNumberish = 0.001): void {
+export function expectEqualWithError(
```

Comment:
Good generalization (closer to parity with forge); we should start using this now.